modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/ServiceConfigurationManager.java

```diff
@@ -31,7 +66,84 @@ public ServiceConfigurationEntity storeAndFlush(
         return serviceConfigurationStorage.storeAndFlush(serviceConfigurationEntity);
     }
 
+    /**
+     * delete service configuration.
+     *
+     * @param serviceConfigurationEntity ServiceConfigurationEntity.
+     */
     public void deleteServiceConfiguration(ServiceConfigurationEntity serviceConfigurationEntity) {
         serviceConfigurationStorage.deleteServiceConfiguration(serviceConfigurationEntity);
     }
+
+
+    /**
+     * update ServiceConfiguration.
+     *
+     * @param serviceId                  The id of the deployed service.
+     * @param serviceConfigurationUpdate serviceConfigurationUpdate.
+     */
+    public UUID changeServiceConfiguration(String serviceId,
+            ServiceConfigurationUpdate serviceConfigurationUpdate) {
+        DeployServiceEntity deployServiceEntity =
+                deployServiceEntityHandler.getDeployServiceEntity(UUID.fromString(serviceId));
+        ServiceConfigurationEntity serviceConfigurationEntity =
+                deployServiceEntity.getServiceConfigurationEntity();
+        ServiceConfigurationUpdateRequest request = initServiceConfigurationUpdateRequest(
+                serviceConfigurationUpdate.getConfiguration());
+        try {
+            validate(deployServiceEntity, serviceConfigurationUpdate);
+            updateServiceConfigurationUpdateStatus(request, ServiceConfigurationStatus.PROCESSING);
```

Comment:
we must set the process status to PENDING and leave it. 
All further updates to the status comes from other  process which runs the configuration update on service VMs.